raiden-cli/src/config.json

```diff
@@ -4,6 +4,20 @@
   "autoSettle": true,
   "autoUDCWithdraw": false,
   "caps": {
-    "Receive": 1
+    "Receive": 1,
+    "Mediate": 1,
```

Comment:
Please, remove trailing commas in objects in JSON; Some JSON parsers complain about this. Ours doesn't, but only for json, it's better to keep compatibility here.

Reply:
✔️